### PR TITLE
Adds UIs for volume and pressure pumps

### PR DESCRIPTION
**Pull request by ike709**
## About the PR 
See https://github.com/space-wizards/space-station-14/issues/4203

Adds a UI for volume pumps and pressure pumps.

**Screenshots**

image

**Changelog**

:cl:
- add: Pressure pumps now have a UI
- add: Volume pumps now have a UI

### CODE REVIEW

**Review by moonheart08 — APPROVED** [review]
lgtm

File: Content.Client/Atmos/UI/GasVolumePumpWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<SS14Window xmlns="https://spacestation14.io"
```
Comment (mirrorcult) [inline Content.Client/Atmos/UI/GasVolumePumpWindow.xaml:1]: there's a lot of duplication between the volume pump and pressure pump but they're both so small that I don't think I really care

File: Content.Server/Atmos/Piping/Binary/EntitySystems/GasPressurePumpSystem.cs [hunk 2]
```diff
@@ -79,5 +89,38 @@ private void OnPumpLeaveAtmosphere(EntityUid uid, GasPressurePumpComponent compo
...
+        private void OnOutputPressureChangeMessage(EntityUid uid, GasPressurePumpComponent pump, GasPressurePumpChangeOutputPressureMessage args)
+        {
+            pump.TargetPressure = Math.Clamp(args.Pressure, 0f, Atmospherics.MaxOutputPressure);
+            DirtyUI(uid, pump);
+
+        }
+
+        private void DirtyUI(EntityUid uid, GasPressurePumpComponent? pump)
+        {
+
```
Comment (mirrorcult) [inline Content.Server/Atmos/Piping/Binary/EntitySystems/GasPressurePumpSystem.cs:118]: whitespace

**Review by mirrorcult — APPROVED** [review]

File: Content.Server/Atmos/Piping/Binary/EntitySystems/GasPressurePumpSystem.cs (hunk 2 above)
Comment (ike709) [inline Content.Server/Atmos/Piping/Binary/EntitySystems/GasPressurePumpSystem.cs:118]:
```suggestion
```

File: Content.Server/Atmos/Piping/Binary/EntitySystems/GasVolumePumpSystem.cs [hunk 3]
```diff
@@ -83,5 +93,39 @@ private void OnVolumePumpUpdated(EntityUid uid, GasVolumePumpComponent pump, Atm
...
+        private void OnTransferRateChangeMessage(EntityUid uid, GasVolumePumpComponent pump, GasVolumePumpChangeTransferRateMessage args)
+        {
+            pump.TransferRate = Math.Clamp(args.TransferRate, 0f, Atmospherics.MaxTransferRate);
+            DirtyUI(uid, pump);
+
+        }
+
+        private void DirtyUI(EntityUid uid, GasVolumePumpComponent? pump)
+        {
+
```
Comment (ike709) [inline Content.Server/Atmos/Piping/Binary/EntitySystems/GasVolumePumpSystem.cs:123]:
```suggestion
```

**Pull request merged**